### PR TITLE
Limit number of attribute threads to reduce thread ping pong

### PR DESCRIPTION
**Pull request by baldersheim**
@geirst PR

### CODE REVIEW

File: searchcore/src/vespa/searchcore/proton/server/threading_service_config.cpp [hunk 1]
```diff
@@ -22,7 +22,7 @@ namespace {
 uint32_t
 calculateIndexingThreads(uint32_t cfgIndexingThreads, double concurrency, const HwInfo::Cpu &cpuInfo)
 {
-    double scaledCores = cpuInfo.cores() * concurrency;
+    double scaledCores = std::min(12.0, cpuInfo.cores() * concurrency);
```
Comment (geirst) [inline searchcore/src/vespa/searchcore/proton/server/threading_service_config.cpp]: Please add a description on why we do the capping. Otherwise ok.

**Comment by geirst** [comment]
@toregge FYI

File: searchcore/src/vespa/searchcore/proton/server/threading_service_config.cpp (hunk 1 above)
Comment (baldersheim) [reply searchcore/src/vespa/searchcore/proton/server/threading_service_config.cpp]: Fixed

**Review by geirst — COMMENTED** [review]
👍

**Pull request merged**